Neos.Flow/Classes/I18n/Parser/DatetimeParser.php

```diff
@@ -78,7 +77,6 @@ public function parseDatetimeWithCustomPattern($datetimeToParse, $format, I18n\L
      * locale.
      *
      * @param string $dateToParse date to be parsed
-     * @param I18n\Locale $locale
```

Comment:
I also find this weird to leave out intermediate arguments in the docblock, just because those are fully typed in code. Yes, this line does not yield additional information, but when you look over the docblock it should be a consistent view and not partial.

Reply:
agreed

Reply:
i like this approach suggested here better: https://discuss.neos.io/t/rfc-updated-default-code-style/5836?u=marc